Brackets/Brackets/main.swift

```diff
@@ -16,7 +16,7 @@ if let expression = readLine()
 }
 
 var bracketsList = Array(inputBrackets)
-var lenBrackets = inputBrackets.count
+let lenBrackets = inputBrackets.count
 var yesesAndNos: [String] = []
 
```

Comment:
Do you need allBrackets ? Instead you can check if it is part of openBrackets or closedBrackets